src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"messages.securitycraft:blockpocket.unowned": "Vous ne possédez pas le bloc \"%2$s\" à la position %1$s!",
+	"messages.securitycraft:blockpocket.activated": "Bloc de poche activé!",
+	"messages.securitycraft:blockpocket.deactivated": "Bloc de poche désactivé!",
+
+	"jei.securitycraft:reinforcedBloc.info": "Clic droit sur un Blindeur de blocs universel et placez le bloc suivant dans l’emplacement. Sinon, vous pouvez placer le bloc suivant vers le bas et le casser avec le Blindeur de blocs universel.",
+
+	"config.waila.plugin_securitycraft": "SecurityCraft",
+	"config.waila.plugin_securitycraft.showowner": "Afficher le propriétaire ?",
+	"config.waila.plugin_securitycraft.showmodules": "Afficher les Modules ?",
+	"config.waila.plugin_securitycraft.showpasswords": "Afficher le mots de passe ?",
+	"config.waila.plugin_securitycraft.showcustomname": "Afficher les blocs nommés ?",
+	"waila.securitycraft:owner": "Propriétaire:",
+	"waila.securitycraft:equipped": "Équipé avec:",
+	"waila.securitycraft:password": "mot de passe:",
+	"waila.securitycraft:password.notSet": "Non renseigné",
+	"waila.securitycraft:customName": "Alias:",
+	"waila.securitycraft:customName.notSet": "Non renseigné",
+
+	"tooltip.securitycraft:mine": "Mine",
+	"tooltip.securitycraft:sentry": "Sentinelle",
+	"tooltip.securitycraft:camera": "Caméra",
+	"tooltip.securitycraft:cameraMonitor": "# des caméras associées:",
+	"tooltip.securitycraft:module.modifiable": "Le module est modifiable!",
+	"tooltip.securitycraft:module.notModifiable": "Le module n’est pas modifiable!",
+	"tooltip.securitycraft:module.playerCustomization.players": "Joueurs",
+	"tooltip.securitycraft:module.itemAddons.usage.blocks": "Vous pouvez ajouter # bloc(s).",
+	"tooltip.securitycraft:module.itemAddons.usage.items": "Vous pouvez ajouter # élément(s).",
+	"tooltip.securitycraft:module.itemAddons.usage.blocksAndItems": "Vous pouvez ajouter # bloc(s) et $ élément(s).",
+	"tooltip.securitycraft:module.itemAddons.added": "Ajouté",
+	"tooltip.securitycraft:keycard.uses": "utilisations restantes:",
+
+	"itemGroup.securitycraft.decoration": "SecurityCraft: Décoration",
+	"itemGroup.securitycraft.explosives": "SecurityCraft: Explosifs",
+	"itemGroup.securitycraft.technical": "SecurityCraft: Technique",
+
+	"module.securitycraft.ims.whitelist_module.description": "L’ajout d’un module de liste blanche à un S.M.I. permettra aux joueurs inscrits sur la liste blanche de pénétrer dans la portée du I.M.S sans être ciblés.",
+	"module.securitycraft.inventory_scanner.whitelist_module.description": "L’ajout d’un module de liste blanche à un scanner d’inventaire permettra aux joueurs sur liste blanche de parcourir le champ de scan sans que leur inventaire soit contrôlé.",
+	"module.securitycraft.inventory_scanner.smart_module.description": "L’ajout d’un module intelligent à un scanner d’inventaire permettra de contrôler les enchantements et les autres changements de métadonnées lors de la numérisation des
```

Comment:
You accidentally changed `Keycard` to `Carte d‘accès` in the language keys here.